src/lib.rs

```diff
@@ -60,23 +58,23 @@ pub mod configuration {
         pub const TURKISH: Self = Self(12852);
     }
 
-    impl<DB> FromSql<Regconfig, DB> for TsConfiguration
+    impl<DB> FromSql<RegConfig, DB> for TsConfiguration
     where
         DB: Backend,
-        i32: FromSql<Integer, DB>,
+        u32: FromSql<Integer, DB>,
     {
-        fn from_sql(bytes: Option<&DB::RawValue>) -> deserialize::Result<Self> {
-            <i32 as FromSql<Integer, DB>>::from_sql(bytes).map(|oid| TsConfiguration(oid as u32))
+        fn from_sql(bytes: RawValue<'_, DB>) -> deserialize::Result<Self> {
+            u32::from_sql(bytes).map(|oid| TsConfiguration(oid))
         }
     }
 
-    impl<DB> ToSql<Regconfig, DB> for TsConfiguration
+    impl<DB> ToSql<RegConfig, DB> for TsConfiguration
     where
         DB: Backend,
-        i32: ToSql<Integer, DB>,
+        u32: ToSql<Integer, DB>,
     {
-        fn to_sql<W: Write>(&self, out: &mut Output<W, DB>) -> serialize::Result {
-            <i32 as ToSql<Integer, DB>>::to_sql(&(*&self.0 as i32), out)
+        fn to_sql<'b>(&'b self, out: &mut Output<'b, '_, DB>) -> serialize::Result {
+            self.0.to_sql(out)
```

Comment:
The old constraint could be used by just doing the following:

```suggestion
        fn to_sql<'b>(&'b self, out: &mut Output<'b, '_, DB>) -> serialize::Result {
            <i32 as ToSql<Integer, DB>>::to_sql(&(*&self.0 as i32), out.reborrow())
```

Reply:
done that, I removed the `DB` generic type and replaced as `Pg` directly (wasn't compile due to a not implemented trait for `Output`), the library is just for Postgres so it was useless using this generics.